pkg/model/bootstrapscript.go

```diff
@@ -39,9 +39,11 @@ import (
 
 // BootstrapScript creates the bootstrap script
 type BootstrapScript struct {
-	NodeUpSource        string
-	NodeUpSourceHash    string
-	NodeUpConfigBuilder func(ig *kops.InstanceGroup) (*nodeup.Config, error)
+	NodeUpSourceAmd64     string
+	NodeUpSourceHashAmd64 string
+	NodeUpSourceArm64     string
+	NodeUpSourceHashArm64 string
```

Comment:
Perhaps these would be cleaner as two maps of arch to string?

Reply:
I thought of that, but in the bootstrap script they are separate vars (bash). This way keeps the same style.

Reply:
This way complicates initializing the struct. They can still be four separate template functions.

Reply:
Done. It looks better now.